charts/datafold/templates/storageclass.yaml

```diff
@@ -26,5 +26,15 @@ allowVolumeExpansion: true
 volumeBindingMode: "WaitForFirstConsumer"
 {{- end }}
 {{- if (eq .Values.global.cloudProvider "azure") }}
-  fail "Storage class not yet implemented for azure." }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "sc-datafold-aws"
+provisioner: "disk.csi.azure.com"
+reclaimPolicy: "Retain"
+parameters:
+  fsType: ext4
+  skuname: StandardSSD_LRS
```

Comment:
Doesn't this need to be configurable? Since it is in terraform-azure-datafold.

Reply:
Ah, sku ?  Op zich wel.  Andere keer, want het is wel de default.